### DISCUSSION TITLE
the libmotioncapture submodule on the master not point(update) to the self main

### DISCUSSION THREAD

**Discussion by duguguang** [USC-ACTLab/crazyswarm#761] [Q&A]
i see the submodule hasn't been updated in two years

**Comment by knmcguire** [comment]
Hi! 

Development is now more focuses on Crazyswarm2 instead of 1, so that is why probably? You should probably try out Crazyswarm2 instead: https://imrclab.github.io/crazyswarm2/

**Comment by duguguang** [comment]
But In this case, users of crazyswarm1 must manually switch the submodule to the main branch each time to use the newly added motion capture types. This is not very user-friendly for beginners, and I have recently received frequent requests for help.
